sparta/test/SpartaSharedPointer/SpartaSharedPointer_test.cpp

```diff
@@ -25,6 +25,20 @@ class MyType
 
 };
 
+class DerivedType : public MyType
+{
+public:
+    DerivedType() :
+        b(20)
+    {}
+
```

Comment:
I'll bet that if you put in a destructor for `DerivedType` and added a count for the number of times it's destructed, it will be 0 after your test.  :wink: